recipes/flakeheaven/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "flakeheaven" %}
+{% set version = "0.11.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/flakeheaven-{{ version }}.tar.gz
+  sha256: 20f5a573b8e85be5a73fed2e3967f7ab8b5e77714a5898d8b634dd31a0b75f9b
+
+build:
+  entry_points:
+    - flake8heavened = flakeheaven:flake8_entrypoint
+    - flakeheaven = flakeheaven:entrypoint
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - poetry
+    - python >=3.0,<4.0
+  run:
+    - colorama
+    - entrypoints
+    - flake8 >=4.0.1,<5.0.0
+    # Only required for [py<38] but always include so this can be noarch: python
+    - importlib-metadata >=1.0,<2.0
+    - pygments
+    - python >=3.0,<4.0
+    - toml
+    - urllib3
+
+test:
+  imports:
+    - flakeheaven
+  commands:
+    - pip check
+    - flake8heavened --help
+    - flakeheaven --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/flakeheaven/flakeheaven
+  summary: FlakeHeaven is a [Flake8](https://gitlab.com/pycqa/flake8) wrapper to make it cool.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - pwoolvett
```

Comment:
@pwoolvett, are you ok with being a conda-forge recipe maintainer for `flakeheaven`? Also, is there anyone else you'd recommend adding as a maintainer?

Reply:
yourself! I do not use conda forge so you would be the best candidate :)

Reply:
Haha sure, I can add myself in, but I would recommend adding in 1 or 2 other devs from https://github.com/flakeheaven/flakeheaven to this list. Most everything is pretty automated on conda-forge, a bot will open a PR here after a PyPI release, and you just need to click a button to merge it (making sure to update any dependencies if they've changed).

Reply:
Yep, @pwoolvett you should also consider using conda ;)

Reply:
@weiji14 You can add me as a maintainer too if you like.

Reply:
Cool, thanks @mcarans, I've added you in at https://github.com/conda-forge/staged-recipes/pull/18471/commits/c18d7f11bd3c451b8dff4538c20d01a7b0b50d4b.

@pwoolvett, just to be sure, you're ok with being a maintainer here too? Need an explicit yes/no answer.

Reply:
Cool, that's ok. We can add you in (or anyone else) next time too, no pressure :grin: